sdk/Lusid.Sdk/Api/TransactionPortfoliosApi.cs

```diff
@@ -37,7 +40,7 @@ public interface ITransactionPortfoliosApi : IApiAccessor
         /// <param name="adjustHoldingRequest">The selected set of holdings to adjust to the provided targets for the              transaction portfolio.</param>
         /// <param name="reconciliationMethods">Optional parameter for specifying a reconciliation method: e.g. FxForward. (optional)</param>
         /// <returns>AdjustHolding</returns>
-        AdjustHolding AdjustHoldings (string scope, string code, DateTimeOrCutLabel effectiveAt, List<AdjustHoldingRequest> adjustHoldingRequest, List<string> reconciliationMethods = null);
+        AdjustHolding AdjustHoldings(string scope, string code, DateTimeOrCutLabel effectiveAt, List<AdjustHoldingRequest> adjustHoldingRequest, List<string> reconciliationMethods = default(List<string>));
```

Comment:
What is the implication of the change from a default value of `null` to `default{type}`?

Reply:
I guess on this line empty query params get added?

```
            if (reconciliationMethods != null)
            {
                localVarRequestOptions.QueryParameters.Add(Lusid.Sdk.Client.ClientUtils.ParameterToMultiMap("multi", "reconciliationMethods", reconciliationMethods));
            }
```

Reply:
Okay, makes sense